x-pack/plugins/security_solution/public/management/components/page_overlay/page_overlay.tsx

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, {
+  memo,
+  ReactNode,
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+  CSSProperties,
+} from 'react';
+import styled, { createGlobalStyle } from 'styled-components';
+import { EuiFocusTrap, EuiPortal } from '@elastic/eui';
+import classnames from 'classnames';
+import { useLocation } from 'react-router-dom';
+import { EuiPortalProps } from '@elastic/eui/src/components/portal/portal';
+import { EuiTheme } from '@kbn/kibana-react-plugin/common';
+import { TIMELINE_OVERRIDES_CSS_STYLESHEET } from '../../../common/components/page';
+import {
+  SELECTOR_TIMELINE_IS_VISIBLE_CSS_CLASS_NAME,
+  TIMELINE_EUI_THEME_ZINDEX_LEVEL,
+} from '../../../timelines/components/timeline/styles';
+import { useIsMounted } from '../../hooks/use_is_mounted';
+
+const OverlayRootContainer = styled.div`
+  border: none;
+
+  display: block;
+  position: fixed;
+  overflow: hidden;
+
+  top: calc((${({ theme: { eui } }) => eui.euiHeaderHeightCompensation} * 2));
```

Comment:
I think I do. I remember playing with this when I first created this component and I could not get the value to come out property unless I used `calc()`.

Reply:
With calc here it renders to `top: calc(96px)`. What we want is `top: 96px` instead. The former works too but I think it is redundant.

Reply:
can you show me a screen capture from the browser with `top: 96px`?

When I make this change localy to: 

```
top: ${({ theme: { eui } }) => eui.euiHeaderHeightCompensation * 2};
```

`top` is not output to the styles (see below). I think it's because the CSS compiler is attempting to do something like `{string} * {number}` and that craps out and does not output the value (maybe).

<img width="358" alt="image" src="https://user-images.githubusercontent.com/56442535/172866723-72f099da-6029-49f6-af9a-6d5cc37b927a.png">